### DISCUSSION TITLE
Move point to point function

### DISCUSSION THREAD

**Discussion by yuxinll** [visose/Robots#203] [Q&A] [answered]
Is there a function that I can move the robot from point to point (Move P function)? There is only move Linear and Joint now in the gh definition.

Thanks so much for your help!

**Comment by visose** [comment]
Hi, see the issue and related discussions found here: https://github.com/visose/Robots/issues/132

**Comment by visose** [comment, marked as answer]
MoveP is now supported for UR as the `Process` motion type. Add the Motion input on Create Target and choose `Process`; for UR this generates `movep(...)`. Closing this as implemented.
